### ISSUE TITLE
Getting a strange giant error when I try to change some loot pools

### ISSUE THREAD

**Issue by MechanosG** [Leviathan143/LootTweaker#15]
When I try to use this small script:

```
# import needed thingies
import loottweaker.vanilla.loot.LootTables;
import loottweaker.vanilla.loot.LootTable;
import loottweaker.vanilla.loot.LootPool;
import loottweaker.vanilla.loot.Conditions;
import loottweaker.vanilla.loot.Functions;

# Immersive Engineering
val IEchest = LootTables.getTable("immersiveengineering:chests/engineers_house");
val IEpool = IEchest.getPool("immersiveengineering:engineers_village_house");

IEpool.removeLootTableEntry("steel_rod");
IEpool.removeLootTableEntry("aluminium_rod");
IEpool.removeLootTableEntry("steel_component");
IEpool.removeLootTableEntry("aluminium_ingot");

# blacksmith chests
val BSchest = LootTables.getTable("minecraft:chests/village_blacksmith");

BSchest.removePool("action_figures");
BSchest.removePool("Ender IO");
```

I get this gigantic error:

```
[14:08:10] [Server thread/INFO] [STDOUT]: [minetweaker.runtime.MTTweaker:load:142]: MineTweaker: Loading lootchanges.zs
[14:08:10] [Server thread/INFO] [STDOUT]: [minetweaker.runtime.MTTweaker:load:142]: MineTweaker: Loading intermod
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: java.lang.NullPointerException
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at java.lang.invoke.DirectMethodHandle.checkBase(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at leviathan143.loottweaker.common.darkmagic.CommonMethodHandles.getEntriesFromPool(CommonMethodHandles.java:91)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at leviathan143.loottweaker.common.zenscript.ZenLootPoolWrapper.applyLootTweaks(ZenLootPoolWrapper.java:153)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at leviathan143.loottweaker.common.zenscript.ZenLootTableWrapper.applyLootTweaks(ZenLootTableWrapper.java:97)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at leviathan143.loottweaker.common.tweakers.loot.LootTableTweaker.applyTweaks(LootTableTweaker.java:70)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at leviathan143.loottweaker.common.tweakers.loot.LootTableTweaker.onTableLoad(LootTableTweaker.java:63)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_581_LootTableTweaker_onTableLoad_LootTableLoadEvent.invoke(.dynamic)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:185)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.event.ForgeEventFactory.loadLootTable(ForgeEventFactory.java:625)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.common.ForgeHooks.loadLootTable(ForgeHooks.java:1073)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraft.world.storage.loot.LootTableManager$Loader.func_186518_c(LootTableManager.java:148)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraft.world.storage.loot.LootTableManager$Loader.load(LootTableManager.java:71)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraft.world.storage.loot.LootTableManager$Loader.load(LootTableManager.java:52)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3524)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2317)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2280)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2195)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.cache.LocalCache.get(LocalCache.java:3934)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3938)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4821)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4827)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraft.world.storage.loot.LootTableManager.func_186521_a(LootTableManager.java:39)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraft.world.storage.loot.LootTableManager.func_186522_a(LootTableManager.java:48)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at leviathan143.loottweaker.common.tweakers.loot.LootTableTweaker$2.handle(LootTableTweaker.java:96)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at leviathan143.loottweaker.common.tweakers.loot.LootTableTweaker$2.handle(LootTableTweaker.java:86)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at minetweaker.util.EventList.publish(EventList.java:49)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at minetweaker.MineTweakerImplementationAPI.reload(MineTweakerImplementationAPI.java:513)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at minetweaker.MineTweakerImplementationAPI.onServerStart(MineTweakerImplementationAPI.java:411)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at minetweaker.mc1102.MineTweakerMod.onServerStarting(MineTweakerMod.java:171)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:616)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:243)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:221)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:145)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.fml.common.Loader.serverStarting(Loader.java:855)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraftforge.fml.common.FMLCommonHandler.handleServerStarting(FMLCommonHandler.java:292)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at Reflector.callBoolean(Reflector.java:700)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraft.server.integrated.IntegratedServer.func_71197_b(IntegratedServer.java:205)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431)
[14:08:10] [Server thread/INFO] [STDERR]: [leviathan143.loottweaker.common.darkmagic.CommonMethodHandles:getEntriesFromPool:95]: 	at java.lang.Thread.run(Unknown Source)
[14:08:10] [Server thread/ERROR] [FML]: Exception caught during firing event net.minecraftforge.event.LootTableLoadEvent@46ccc5f8:
java.lang.NullPointerException
	at java.lang.invoke.DirectMethodHandle.checkBase(Unknown Source) ~[?:1.8.0_101]
	at leviathan143.loottweaker.common.darkmagic.CommonMethodHandles.getEntriesFromPool(CommonMethodHandles.java:91) ~[CommonMethodHandles.class:?]
	at leviathan143.loottweaker.common.zenscript.ZenLootPoolWrapper.applyLootTweaks(ZenLootPoolWrapper.java:153) ~[ZenLootPoolWrapper.class:?]
	at leviathan143.loottweaker.common.zenscript.ZenLootTableWrapper.applyLootTweaks(ZenLootTableWrapper.java:97) ~[ZenLootTableWrapper.class:?]
	at leviathan143.loottweaker.common.tweakers.loot.LootTableTweaker.applyTweaks(LootTableTweaker.java:70) ~[LootTableTweaker.class:?]
	at leviathan143.loottweaker.common.tweakers.loot.LootTableTweaker.onTableLoad(LootTableTweaker.java:63) ~[LootTableTweaker.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_581_LootTableTweaker_onTableLoad_LootTableLoadEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:185) [EventBus.class:?]
	at net.minecraftforge.event.ForgeEventFactory.loadLootTable(ForgeEventFactory.java:625) [ForgeEventFactory.class:?]
	at net.minecraftforge.common.ForgeHooks.loadLootTable(ForgeHooks.java:1073) [ForgeHooks.class:?]
	at net.minecraft.world.storage.loot.LootTableManager$Loader.func_186518_c(LootTableManager.java:148) [bas$a.class:?]
	at net.minecraft.world.storage.loot.LootTableManager$Loader.load(LootTableManager.java:71) [bas$a.class:?]
	at net.minecraft.world.storage.loot.LootTableManager$Loader.load(LootTableManager.java:52) [bas$a.class:?]
	at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3524) [guava-17.0.jar:?]
	at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2317) [guava-17.0.jar:?]
	at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2280) [guava-17.0.jar:?]
	at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2195) [guava-17.0.jar:?]
	at com.google.common.cache.LocalCache.get(LocalCache.java:3934) [guava-17.0.jar:?]
	at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3938) [guava-17.0.jar:?]
	at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4821) [guava-17.0.jar:?]
	at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4827) [guava-17.0.jar:?]
	at net.minecraft.world.storage.loot.LootTableManager.func_186521_a(LootTableManager.java:39) [bas.class:?]
	at net.minecraft.world.storage.loot.LootTableManager.func_186522_a(LootTableManager.java:48) [bas.class:?]
	at leviathan143.loottweaker.common.tweakers.loot.LootTableTweaker$2.handle(LootTableTweaker.java:96) [LootTableTweaker$2.class:?]
	at leviathan143.loottweaker.common.tweakers.loot.LootTableTweaker$2.handle(LootTableTweaker.java:86) [LootTableTweaker$2.class:?]
	at minetweaker.util.EventList.publish(EventList.java:49) [EventList.class:?]
	at minetweaker.MineTweakerImplementationAPI.reload(MineTweakerImplementationAPI.java:513) [MineTweakerImplementationAPI.class:?]
	at minetweaker.MineTweakerImplementationAPI.onServerStart(MineTweakerImplementationAPI.java:411) [MineTweakerImplementationAPI.class:?]
	at minetweaker.mc1102.MineTweakerMod.onServerStarting(MineTweakerMod.java:171) [MineTweakerMod.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_101]
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:616) [FMLModContainer.class:?]
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_101]
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) [guava-17.0.jar:?]
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) [guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) [guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) [guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:275) [guava-17.0.jar:?]
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:243) [LoadController.class:?]
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:221) [LoadController.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_101]
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) [guava-17.0.jar:?]
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) [guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) [guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) [guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:275) [guava-17.0.jar:?]
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:145) [LoadController.class:?]
	at net.minecraftforge.fml.common.Loader.serverStarting(Loader.java:855) [Loader.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.handleServerStarting(FMLCommonHandler.java:292) [FMLCommonHandler.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_101]
	at Reflector.callBoolean(Reflector.java:700) [Reflector.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71197_b(IntegratedServer.java:205) [bzl.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:431) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_101]
```
Which ends up breaking a bunch of other things, resulting in MC permanently freezing on the Loading world Building terrain screen.

Commenting out the script, or removing it, fixes the problem.  I tested commenting out the blacksmith chest section first, and it still errored.  So whatever the problem is, must be in the Immersive Engineering section, or the Imports.  Dunno if I'm at fault or not.  Maybe the weird immersiveengineering:engineers_village_house pool name did it?  Or maybe I used removeLootTableEntry incorrectly?

MC 1.10.2
LootTweaker 0.0.5.1
CraftTweaker 3.0.24

**Comment by Leviathan143** [comment]
>Or maybe I used removeLootTableEntry incorrectly?

Yes, you are using it wrong. removeLootTableEntry is for removing references to other loot tables from a loot table. To remove items, you need to use removeItemEntry.

However I would still like you to keep this issue open as LootTweaker should provide helpful error messages rather than stacktraces most of the time.

**Comment by Leviathan143** [comment]
Closing as 6ae9796 & 8e046fe should have fixed this issue on 1.10.2 and 1.11.2 respectively. If the issue still occurs, comment on this issue and it'll be reopened.
